.github/workflows/ci.yml

```diff
@@ -5,6 +5,17 @@ jobs:
   job1:
     name: trunner.nim
     runs-on: ubuntu-latest
+    container: nimlang/nim:latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Compile and run `tests/trunner.nim`
+      run: "nim c -r --styleCheck:error --hint[Processing]:off tests/trunner.nim"
```

Comment:
What is the purpose of testing outside the container?
To split the concerns and test the parts separately?

Reply:
Advantages:
- Faster feedback from CI: the main tests (to which we will add in the future) finish in ~30 seconds total, not 4 minutes. (I think it's okay if we run the `exercism/nim` test only inside the container). And the GitHub UI more immediately shows the cause of failure if there are 2 checks and only one of them has a nice green checkmark.
- Test the parts separately: It ensures that we don't do something strange that makes the tests only work in a container, and not outside.

Reply:
👍